CHANGELOG.md

```diff
@@ -6,8 +6,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Notes
 - This release supports for API V800,V1000 and V1200 for HypervisorManager resource
 - Added resource files for Server Certificate supporting API V800, V1000, V1200
+- Added resource files for Hypervisor Cluster Profile supporting API V800, V1000, V1200
```

Comment:
You can combine the notes description together and point to the resources supported to the list below. You can add extra comments only if there a feature different from the existing notes.

Reply:
Done